IBAnimatableApp/TransitionTableViewController.swift

```diff
@@ -44,47 +44,168 @@ class TransitionTableViewController: UITableViewController {
 
 private extension TransitionTableViewController {
 
+  // swiftlint:disable line_length
```

Comment:
That's depending on your screen size, the long lines are generally hard to read on mine 😬